ui/shared/polyfill.number.toLocaleString.js

```diff
@@ -0,0 +1,294 @@
+// polyfill.number.toLocaleDateString
```

Comment:
I think this file fits more in `imports` IMO. Also, I'd only import it in `Utils` like I said above